Release/1.1.0 --> master

ludwinggsanchez opened the pull request:
Revisar si falta algo en el SQL adjunto en #93 (y validar)

ludwinggsanchez committed:
feat udistrital/arka_cliente#874: Depreciacion

- Migraciones para manejo separado de la amortizacion y depreciacion

ludwinggsanchez committed:
feat udistrital/arka_cliente#874: Mediciones posteriores

Co-authored-by: Alex Bustos <AlexFBP@users.noreply.github.com>

AlexFBP committed:
management: Merge pull request #105 from udistrital/feature/Amortizacion

feat udistrital/arka_cliente#874: Depreciacion

AlexFBP committed:
management: Merge pull request #106 from udistrital/release/1.0.0

Release/1.0.0

ludwinggsanchez committed:
docs: v1.1.0

AlexFBP committed:
devops: fix: ignorar fallo al notificar por telegram

AlexFBP reviewed (commented):
VoBo de #109, #107 y #106. Pendiente por VoBo final de Arquitectura para pasar a produccion

ludwinggsanchez committed:
feat udistrital/arka_cliente#854: Hallazgos

- Se omiten registros en tabla subgrupo_subgrupo para construir árbol a nivel de elemento/producto.

ludwinggsanchez committed:
feat udistrital/arka_cliente#854: Hallazgos

- Validaciones adicionales algoritmo para asignar cuentas contables.

AlexFBP committed:
fix: auditoria actualizada

AlexFBP commented:
Solicitud de actualización de DB en prod en https://tuleap.portaloas.udistrital.edu.co/plugins/tracker/?aid=47376

AlexFBP commented:
Faltaba la siguiente migración en producción:
https://github.com/udistrital/catalogo_elementos_crud/blob/develop/database/scripts/20220222_221225_amortizacion_up.sql
Solicitada en el kanban:
https://tuleap.portaloas.udistrital.edu.co/plugins/tracker/?aid=47748
